ingest.py

```diff
@@ -5,24 +5,38 @@
 from langchain.embeddings import OpenAIEmbeddings
 from langchain.text_splitter import RecursiveCharacterTextSplitter
 from langchain.vectorstores.faiss import FAISS
-
+from langchain.document_loaders import RecursiveUrlLoader
+import numpy as np
 
 def ingest_docs():
     """Get documents from web pages."""
-    loader = ReadTheDocsLoader("langchain.readthedocs.io/en/latest/")
+    loader = ReadTheDocsLoader("api.python.langchain.com/en/latest", features="lxml")
+    loader2 = ReadTheDocsLoader("python.langchain.com", features="lxml")
+    # loader2 = RecursiveUrlLoader(url="https://python.langchain.com/docs/get_started/introduction", max_depth=2, extractor=lambda x: Soup(x, "lxml").text)
+
     raw_documents = loader.load()
+    print(len(raw_documents))
+    raw_documents += loader2.load()
+    print(len(raw_documents))
     text_splitter = RecursiveCharacterTextSplitter(
         chunk_size=1000,
         chunk_overlap=200,
     )
     documents = text_splitter.split_documents(raw_documents)
-    embeddings = OpenAIEmbeddings()
+    print(len(documents))
+    embeddings = OpenAIEmbeddings(chunk_size=100)
+    print(embeddings)
     vectorstore = FAISS.from_documents(documents, embeddings)
-
+    
+    # Ensure all embeddings have the same length
```

Comment:
this is very weird - do we know why this is happening?

Reply:
fixed in recent push